analyzers/copyboundary/copy_boundary.go

```diff
@@ -47,54 +47,90 @@ func newInspector(pass *analysis.Pass) *copyBoundaryInspector {
 	return inspectors
 }
 
-func (i *copyBoundaryInspector) inspectAssignStatement(node ast.Node) bool {
+func (c *copyBoundaryInspector) inspectAssignStatement(node ast.Node) bool {
 	assignStmt, ok := node.(*ast.AssignStmt)
 	if !ok {
 		return true
 	}
 
-	typ := checkType(assignStmt.Lhs)
-	if typ == "" {
-		return true
-	}
+	flag := false
+	for i := 0; i < len(assignStmt.Rhs); i++ {
+		lhs := assignStmt.Lhs[i]
 
-	typ = checkType(assignStmt.Rhs)
-	if typ == "" {
-		return true
+		ident, ok := lhs.(*ast.Ident)
+
+		if !ok || ident.Obj == nil || ident.Name == "_" {
+			continue
+		}
+
+		ok = isSliceOrMap(assignStmt.Rhs[i])
+		if !ok {
+			continue
+		}
+
+		flag = ok
```

Comment:
You can print multiple messages for bellow code
```
anotherMap, anotherSlice := newMap, newSlice
```

Use token position `lhs.Pos()` instead of `assignStmt.Pos()` in loop